src/app/containers/app-header/app-header.component.html

```diff
@@ -9,3 +9,4 @@
   [logoIsUsed]="logoIsUsed"
   (navigate)="onNavigate($event)">
 </exui-header>
+<xuilib-service-messages [userRoles]="userRoles" [featureToggleKey]="featureToggleKey" [serviceMessageCookie]="serviceMessageCookie"></xuilib-service-messages>
```

Comment:
I expect this to fail linting as it needs a line at the end.

Reply:
I thought so too, however it passed linting.